smart-connector/src/main/java/eu/knowledge/engine/smartconnector/impl/ReasonerProcessor.java

```diff
@@ -569,4 +599,126 @@ public CompletableFuture<Void> handle(eu.knowledge.engine.reasoner.api.BindingSe
 	public ReasonerPlan getReasonerPlan() {
 		return this.reasonerPlan;
 	}
+	
+	/**
+	 * Returns the knowledge gap of this reasoning node. A knowledge gap is a subset
+	 * of this node's antecedent triple patterns that do not match any neighbor that
+	 * has no knowledge gaps.
+	 * 
+	 * Currently, this method does not show how the knowledge gaps influence each
+	 * other. Some knowledge gaps might have an {@code or}-relation (namely those
+	 * that occur on the same triple) and some might have {@code and}-relations
+	 * (i.e. those that do not occur on the same triple). This information is
+	 * important if you want to know how to solve the gaps because 2 gaps related by
```

Comment:
I don't fully understand my own comments above 😏 . I think we should just remove this whole paragraph, because it is explained in the `@return ...` part below quite clearly👍 . What do you think?

Reply:
Agreed and done